PactSafe iOS Demo/Controllers/LoginWithCheckboxViewController.swift

```diff
@@ -33,9 +32,8 @@ class LoginWithCheckboxViewController: UIViewController {
   private func checkStatus(signerId: String, password: String) {
     ps.signedStatus(for: signerId, groupKey: psGroupKey) { needsAcceptance, contractIds in
       if needsAcceptance {
-        let psAcceptanceVc = PSAcceptanceViewController(self.psGroupKey, signerId: signerId, contractIds: contractIds, customData: nil)
-
         DispatchQueue.main.async {
+          let psAcceptanceVc = PSAcceptanceViewController(self.psGroupKey, signerId: signerId, contractIds: contractIds, customData: nil)
```

Comment:
Just curious, since this was moved around, what does this do?

Reply:
Dispatches the UI on the main thread rather than any background threads!